packages/core/components/AnnotationFilterForm/index.tsx

```diff
@@ -30,17 +41,39 @@ interface AnnotationFilterFormProps {
 export default function AnnotationFilterForm(props: AnnotationFilterFormProps) {
     const dispatch = useDispatch();
     const allFilters = useSelector(selection.selectors.getFileFilters);
+    const fuzzyFilters = useSelector(selection.selectors.getFuzzyFilters);
+    const includeFilters = useSelector(selection.selectors.getIncludeFilters);
+    const excludeFilters = useSelector(selection.selectors.getExcludeFilters);
     const annotationService = useSelector(interaction.selectors.getAnnotationService);
     const [annotationValues, isLoading, errorMessage] = useAnnotationValues(
         props.annotation.name,
         annotationService
     );
 
+    // Check if annotation name is already listed in state as an INCLUDE, EXCLUDE, FUZZY, or regular file filter
+    const usingIncludeFilter: boolean = React.useMemo(() => {
+        return !!includeFilters?.some((filter) => filter.annotationName === props.annotation.name);
+    }, [includeFilters, props.annotation]);
+    const usingExcludeFilter: boolean = React.useMemo(() => {
```

Comment:
I think the language of include and exclude is a bit ambiguous here since we might add the ability to exclude certain annotation values in the future. IMO I think `isExcludingAllAnnotationValues` & `isIncludingAllAnnotationValues` or something along those lines would be clearer.